libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -303,31 +303,31 @@ void AC_Avoid::adjust_velocity(Vector3f &desired_vel_cms, bool &backing_up, floa
 * Limit acceleration so that change of velocity output by avoidance library is controlled
 * This helps reduce jerks and sudden movements in the vehicle
 */
-void AC_Avoid::limit_accel(const Vector3f &original_vel, Vector3f &modified_vel, float dt)
+void AC_Avoid::limit_accel_neu_cm(const Vector3f &original_vel_neu_cms, Vector3f &modified_vel_neu_cms, float dt)
```

Comment:
should, "limit_accel_neu_cm" be "limit_accel_neu_cmss"?  Ah, maybe not because we're dealing with a mix of accelerations and velocities

Reply:
The pattern at the moment is we only add the seconds if it is only a single variable or all the variables are the same unit, in this case accel_neu_cmss. Because we have velocity as well we keep it as just _neu_cm, ie axis and base units.

However, the pattern is capitalisation for functions....